src/plone/restapi/services/linkintegrity/configure.zcml

```diff
@@ -0,0 +1,17 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    xmlns:zcml="http://namespaces.zope.org/zcml"
+    >
+
+  <include package="Products.CMFEditions" />
+
+  <plone:service
+      method="POST"
+      factory=".post.LinkintegrityPOST"
+      for="Products.CMFPlone.interfaces.IPloneSiteRoot"
+      permission="CMFEditions.AccessPreviousVersions"
```

Comment:
Not sure if CMFEditions is the right permission here.

Reply:
Maybe we want to use the permission for accessing folder contents information here (because that's currently the only way to delete items in Volto). The linkintegrity endpoint does not actually delete anything.

Reply:
linkintegrity does not relate to CMFEditions in any way, right? So another permission is needed.

Reply:
Now i changed to **zope2.AccessContentsInformation**. If it's still not ok, let me know which one is better ;)

Reply:
IIRC the `zope2` prefix is discontinued and its `zope.AccessContentInformation`.